### PR TITLE
feat: add about page

### PR DESCRIPTION
**Pull request by lamartire**

### CODE REVIEW

File: components/About/PurpleBlock.vue [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    <h2 class="purple-block-title">
+      <slot />
+    </h2>
+    <div class="purple-block-buttons">
+      <v-button-link>Partner with us</v-button-link>
+      <v-button-link skin="ghost">Join our team</v-button-link>
+    </div>
+
+    </div>
+      </div>
```
Comment (kazurus) [inline components/About/PurpleBlock.vue]: format

File: components/About/TwoColsBlock.vue [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<template>
+  <div v-bind="$attrs">
```
Comment (kazurus) [inline components/About/TwoColsBlock.vue]: v-bind attrs - default behavior

File: components/PageHeader.vue [hunk 3]
```diff
@@ -1,89 +1,199 @@
...
+      width="24"
+      height="20"
+      @click="onTriggerClick"
+    ></v-icon-button>
+    <section class="page-header-menu">
+      <div class="page-header-logo">
+        <v-logo></v-logo>
+      </div>
+      <nav v-if="false" class="page-header-menu-list">
+        <li><a href="#">Pricing</a></li>
```
Comment (kazurus) [inline components/PageHeader.vue]: isn't nuxt-link necessary?

File: components/PageHeader.vue [hunk 4]
```diff
@@ -1,89 +1,199 @@
...
+  methods: {
+    onTriggerClick() {
+      this.isMenuVisible = !this.isMenuVisible;
+    },
+
+    onScreenResize() {},
+
+    debouncedScreenResizeHandler() {
+      debounce(150, () => {
+        console.log(this);
```
Comment (kazurus) [inline components/PageHeader.vue]: artefact?

File: components/PageHeader.vue [hunk 5]
```diff
@@ -1,89 +1,199 @@
...
+      }
+
+      return "close";
+    }
+  },
+
+  watch: {
+    isMenuVisible(val) {
+      if (val) {
+        window.addEventListener("resize", this.debouncedScreenResizeHandler);
```
Comment (kazurus) [inline components/PageHeader.vue]: as I can see, need remove listener on before onmount?

File: components/common/VButtonLink.vue [hunk 6]
```diff
@@ -1,9 +1,20 @@
 <template>
-  <a v-bind="$attrs" class="v-button">
+  <a v-bind="$attrs" :class="{ 'v-button': true, [skin]: !!skin }">
```
Comment (kazurus) [inline components/common/VButtonLink.vue]: v-bind attrs by default

File: pages/index.vue [hunk 7]
```diff
@@ -85,8 +85,8 @@
         </p>
       </template>
       <template slot="list-annotation">
-
-      This means:</template>
+        This means:</template
```
Comment (kazurus) [inline pages/index.vue]: format

**Comment by kazurus** [comment]
@lamartire need to show a menu with existing pages

File: components/PageHeader.vue (hunk 3 above)
Comment (lamartire) [reply components/PageHeader.vue]: No, it's ok

**Pull request merged**